tests/profiler/test_profiler.py

```diff
@@ -387,11 +387,11 @@ def test_pytorch_profiler_trainer_ddp(tmpdir, pytorch_profiler):
         devices=2,
     )
     trainer.fit(model)
-    expected = {"[Strategy]DDPStrategy.validation_step"}
+    expected = {"[pl][profile][Strategy]DDPStrategy.validation_step"}
```

Comment:
This is just for pytorch profiler and will not impact other profilers.

Reply:
Because we need to recognize which tags is from Lightning (There are still other tags from pytorch internal) and we will remove these prefixes in the final view which showed to user in Pytorch Profiller.